guides/common/modules/proc_excluding-hosts-from-rh-cloud-and-insights-client-reports.adoc

```diff
@@ -4,16 +4,18 @@
 You can set the `host_registration_insights` parameter to *False* to omit _rh-cloud_ and _insights-client_ reports.
 {Project} will exclude the hosts from _rh-cloud_ reports and block _insights-client_ from uploading a report to the cloud.
 
-Use the following procedure to change the value of `host_registration_insights` parameter:
+Use the following procedure to change the value of `host_registration_insights` parameter.
+Setting it to *False* prevents new reports from being uploaded if they are associated with the entity.
```

Comment:
There are three places where this module describes setting the parameter to "false": line 8, line 14, and line 16. It would be good to consolidate the information in one place. At first glance, I'd recommend either around line 16 or as part of the step on line 14.

Reply:
Removed this line.